### DISCUSSION TITLE
What is the plan and did we decide to fully move forward here?

### DISCUSSION THREAD

**Discussion by thabaum** [ADefWebserver/ADefWebserver.Module.HtmlTextV2#28] [Q&A]
After our developer meeting and other recent discussions along with merged changes to the `dev` branch of the Oqtane Framework repository it sounds like we should hit the brakes here until things are ironed out there with the new enhancements relating.

The file manager seems like it can be worked out along with the CSS issues among a few other bugs potentially introduced to future looking changes.

We need to get a handle on the new `RichTextEditor` provider details for implementing our new editor.

I feel it would be nice to have a variety of editors for different use cases potentially.  This can be one flavor if we do in fact produce something worthy for production use.

Busy summer so I will try my best to get caught up to speed and help iron out any framework issues by testing over the next month once I see the discussion, maybe a blog article about this one topic.

I will work on this more in the near future if I can and take it to the next step of functionality if this seems like a thing to do as it needs ironed out as well.

The question: 

Are we the community planning/wanting to move forward with this new type of Oqtane Framework rich text editor?  

If so... lets get on page as to what we will do to get this project up to speed with the latest enhancements to the framework itself relating.  I think making the image selector and css or code fixes as functions seem to all work.  Issue is users are unable to select things not properly displayed to the user.  

I did update most all the source to latest Radzen which did introduce some issues between the repository files this HTML Editor originated from of the stripped down component back a couple years ago.  I seen some more new commits recently as well I can review including to further enhance things relating.

Any feedback is appreciated.

**Comment by ADefWebserver** [comment]
When the Oqtane RichTextEditor provider specs come out lets review them and do a call?
